Octokit/Clients/CodesOfConductClient.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    /// <summary>
+    /// A client for GitHub's Code of Conduct API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/codes_of_conduct/">API documentation</a> for more information.
+    /// </remarks>
+    public class CodesOfConductClient : ApiClient, ICodesOfConductClient
+    {
+        /// <summary>
+        /// Instantiates a new GitHub Code of Conduct API client.
+        /// </summary>
+        /// <param name="apiConnection">an API connection</param>
+        public CodesOfConductClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        { }
+
+        /// <summary>
+        /// Gets all code of conducts on GitHub.
+        /// </summary>
+        /// <remarks>See the <a href="https://developer.github.com/v3/codes_of_conduct/#list-all-codes-of-conduct">API documentation</a> for more information.</remarks>
+        /// <returns>A <see cref="IReadOnlyList{CodeOfConduct}"/> on GitHub.</returns>
+        [Preview("scarlet-witch")]
+        [ManualRoute("GET", "/codes_of_conduct")]
+        public Task<IReadOnlyList<CodeOfConduct>> GetAll()
```

Comment:
Because this API doesn't support pagination details, you can add this attribute to get one of the convention tests passing:

```suggestion
        [ExcludeFromPaginationApiOptionsConventionTest("Pagination not supported by GitHub API (tested 14/04/2020)")]
        public Task<IReadOnlyList<CodeOfConduct>> GetAll()
```